src/gui/uithemecommon.h

```diff
@@ -134,6 +134,8 @@ inline QSet<QString> defaultUIThemeIcons()
         u"network-connect"_s,
         u"network-server"_s,
         u"object-locked"_s,
+        u"pause-session"_s,
+        u"paused"_s,
```

Comment:
I guess you just saved these icons for future use, right?

Reply:
Yes. I can delete them or one of them if you want.

Reply:
> I can delete them or one of them if you want.

I'm still hoping to provide a "session pause" feature, so they might come in handy then.